o8g/Scripts/actions.py

```diff
@@ -1216,7 +1216,14 @@ def markerChanged(args):
         inMythosPhase = True
     
     if card.Type == "Agenda" and args.marker == Doom[0] and inMythosPhase == True and card.properties[Doom[0]] != "":
-        if card.markers[Doom] >= int(card.properties[Doom[0]]):
+        maxDoom = int(card.properties[Doom[0]])
+        agendaId = card._id
+        totalDoom = 0
+        for card in table:
+            if card.markers[Doom] is not None:
+                totalDoom = totalDoom + card.markers[Doom]
+        card = Card(agendaId)
```

Comment:
This line and line 1220 are kinda redundant. Why not just use another variable name in the for-loop?

Reply:
rework ;)